### PR TITLE
Replaced concatenated strings with template literals in the plugins/avg.js

### PR DESCRIPTION
**Pull request by stefanlw**
Changes proposed in this pull request:
- Replaced concatenated strings with template literals in the plugins/avg.js

Checklist:
- [x] [Changes](https://github.com/haraka/Haraka/blob/master/Changes.md) updated

### CODE REVIEW

File: plugins/avg.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ exports.hook_data_post = function (next, connection) {
                     if (code !== '220') {
                         // Error
                         connection.results.add(plugin, {
-                            err: 'Unrecognized response: ' + line,
+                            err: `Unrecognized response: '${line}`,
```
Comment (msimerson) [inline plugins/avg.js]: Oops

File: plugins/avg.js [hunk 2]
```diff
@@ -121,9 +121,9 @@ exports.hook_data_post = function (next, connection) {
                     break;
                 case 'scan': {
                     const elapsed = Date.now() - start_time;
-                    connection.loginfo(plugin, 'time=' + elapsed + 'ms ' +
-                                    'code=' + code + ' ' +
-                                    'response="' + response.join(' ') + '"');
+                    connection.loginfo(plugin, `time= ${elapsed} ms 
+                                     code= ${code} 
+                                     response="${response.join(' ')}"`);
```
Comment (msimerson) [inline plugins/avg.js]: White space errors

File: plugins/avg.js [hunk 3]
```diff
@@ -121,9 +121,9 @@ exports.hook_data_post = function (next, connection) {
                     break;
                 case 'scan': {
                     const elapsed = Date.now() - start_time;
-                    connection.loginfo(plugin, 'time=' + elapsed + 'ms ' +
-                                    'code=' + code + ' ' +
-                                    'response="' + response.join(' ') + '"');
+                    connection.loginfo(plugin, `time= ${elapsed} ms\
```
Comment (msimerson) [inline plugins/avg.js]: there are still extra spaces here.

File: plugins/avg.js [hunk 4]
```diff
@@ -121,9 +121,9 @@ exports.hook_data_post = function (next, connection) {
                     break;
                 case 'scan': {
                     const elapsed = Date.now() - start_time;
-                    connection.loginfo(plugin, 'time=' + elapsed + 'ms ' +
-                                    'code=' + code + ' ' +
-                                    'response="' + response.join(' ') + '"');
+                    connection.loginfo(plugin, `time= ${elapsed} ms\ 
+                    code= ${code}\
```
Comment (msimerson) [inline plugins/avg.js]: and incorrect whitespace here.

Comment (stefanlw) [reply plugins/avg.js]: @msimerson I'm not sure what the best option is here? It's too long for one line.

Comment (KingNoosh) [reply plugins/avg.js]: You can actually turn this into an object, the logger will turn it into key=value

Comment (stefanlw) [reply plugins/avg.js]: @KingNoosh Could I have an example? (Sorry, I'm fairly new to this feature and using it as a learning experience)

Comment (msimerson) [reply plugins/avg.js]: You can have a long line (it's not a cardinal sin any longer) or convert it to key-value pairs as @KingNoosh alluded:

```js
connection.loginfo(plugin, {
    time: `${elapsed}ms`,
    code: code,
    response: `"${response.join(' ')}"`,
})
```

**Pull request merged**